lib/assets/components/explore_tiles.dart

```diff
@@ -320,67 +321,119 @@ class ExploreNewsArticleTile extends ExploreTile {
         super(key: key);
 
   Widget buildBody(BuildContext context) {
-    return AspectRatio(
-      aspectRatio: 0.8,
-      child: InkWell(
-        onTap: () => launchLink(article.link),
-        child: Column(
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            AspectRatio(
-              aspectRatio: 1.8,
-              // TODO ink animation doesn't cover image
-              child: CustomNetworkImage(
-                article.headerImage.url,
-                fit: BoxFit.cover,
-              ),
-            ),
-            Expanded(
-              child: Padding(
-                padding: const EdgeInsets.all(8),
+    return BlocProvider(
+      create: (_) => ExploreNewsArticleTileBloc(
+        newsArticle: article,
+        causesService: locator<CausesService>(),
+      ),
+      child:
+          BlocListener<ExploreNewsArticleTileBloc, ExploreNewsArticleTileState>(
+        listener: (context, state) async {
+          if (state is ExploreNewsArticleTileLaunchingState) {
+            await launchUrl(article.link);
+            context.read<ExploreNewsArticleTileBloc>().markLaunched();
+          }
+        },
+        child: BlocBuilder<ExploreNewsArticleTileBloc,
+            ExploreNewsArticleTileState>(
+          builder: (context, state) {
+            return AspectRatio(
+              aspectRatio: 0.8,
+              child: InkWell(
+                onTap: context
+                    .read<ExploreNewsArticleTileBloc>()
+                    .launchNewsArticle,
                 child: Column(
-                  mainAxisAlignment: MainAxisAlignment.spaceBetween,
                   crossAxisAlignment: CrossAxisAlignment.start,
                   children: [
-                    Text(
-                      article.title,
-                      style: Theme.of(context).textTheme.headlineMedium!,
-                      textScaler: const TextScaler.linear(.7),
-                      maxLines: 2,
-                      overflow: TextOverflow.ellipsis,
+                    Container(
+                      width: double.infinity,
+                      constraints: const BoxConstraints(maxHeight: 150),
+                      child: AspectRatio(
+                        aspectRatio: 1.8,
+                        // TODO ink animation doesn't cover image
+                        child: Stack(
+                          alignment: Alignment.topRight,
+                          children: [
+                            CustomNetworkImage(
+                              article.headerImage.url,
+                              fit: BoxFit.cover,
+                              width: double.infinity,
+                              height: 150,
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.all(8.0),
+                              child: BlocBuilder<UserProgressBloc,
+                                  UserProgressState>(
```

Comment:
Wouldn't this fit in one line?
```suggestion
                              child: BlocBuilder<UserProgressBloc, UserProgressState>(
```

Reply:
Yes but when I run format it reformats it to this for some reason. Do you think we need to increase the max line length?